web-client/src/presenter/computeds/AdvancedSearch/practitionerSearchHelper.ts

```diff
@@ -29,9 +29,10 @@ type PractitionerSearchHelperResult = {
   showStateSelect?: boolean;
```

Comment:
Hey I see why these are optional parameters but I think, it is much easier to understand if a function always returns a consistent shape rather than having to null check every value. Could you refactor this helper to have no optional return values, so that we know the properties will always exist?

For reference: web-client/src/presenter/computeds/customCaseReportHelper.ts

Reply:
[Let's consolidate these two conversations here!](https://github.com/ustaxcourt/ef-cms/pull/4951#pullrequestreview-2069139559)